File: src/main/java/com/sudosoo/takeiteasy/service/EventService.java
package com.sudosoo.takeiteasy.service;

import com.sudosoo.takeiteasy.dto.coupon.CouponIssuanceRequestDto;
import com.sudosoo.takeiteasy.dto.event.CreateEventRequestDto;
import com.sudosoo.takeiteasy.dto.event.EventResponseDto;
import org.springframework.transaction.annotation.Transactional;

public interface EventService {

    EventResponseDto createdEvent(CreateEventRequestDto requestDto);

    void couponIssuance(CouponIssuanceRequestDto requestDto);

}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/service/EventService.kt
package com.sudosoo.takeItEasy.service

import com.sudosoo.takeiteasy.dto.coupon.CouponIssuanceRequestDto

interface EventService {
    fun createdEvent(requestDto: CreateEventRequestDto): EventResponseDto

    fun couponIssuance(requestDto: CouponIssuanceRequestDto)
}
